all-actors/src/main/java/org/sunbird/NotificationValidator.java

```diff
@@ -0,0 +1,71 @@
+package org.sunbird;
+
+
+import org.sunbird.pojo.NotificationMode;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.message.IResponseMessage;
+import org.sunbird.message.ResponseCode;
+import org.sunbird.pojo.NotificationRequest;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+
+/**
+ * Validates send notification api request
+ */
+public class NotificationValidator {
+
+
+    public static void validate(NotificationRequest notificationRequest) throws BaseException {
+        validateModeType(notificationRequest.getMode());
+        validateIds(notificationRequest.getIds());
+        //for checking mandatory params of string type
+        checkMandatoryParamsPresent(notificationRequest.getDeliveryType(), JsonKey.DELIVERY_TYPE);
+    }
+
+
+    /**
+     * validates mandatory params of type string
+     *
+     * @param param
+     * @param key
+     * @throws BaseException
+     */
+    private static void checkMandatoryParamsPresent(String param, String key) throws BaseException {
+        if (StringUtils.isEmpty(param)) {
```

Comment:
check for StringUtils.isBlank instead of StringUtils.isEmpty, as isEmpty will return true for "  " string.